src/main/scala/com/databricks/spark/redshift/Conversions.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2015 TouchType Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.databricks.spark.redshift
+
+import java.sql.Timestamp
+import java.text.{DateFormat, FieldPosition, ParsePosition, SimpleDateFormat}
+import java.util.Date
+
+import scala.util.parsing.combinator.JavaTokenParsers
+
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{DataFrame, Row, SQLContext}
+
+/**
+ * Simple parser for Redshift's UNLOAD bool syntax
+ */
+private object RedshiftBooleanParser extends JavaTokenParsers {
```

Comment:
This seems like a really roundabout way to turn the strings `t` and `f` into `true` and `false`.  Why is this better than a simple `if` statement?